src/pages/manager/CustomersListPage.js

```diff
@@ -3,9 +3,63 @@ const { expect } = require('@playwright/test');
 export class CustomersListPage {
   constructor(page) {
     this.page = page; 
+    this.lastRowLocator = page.getByRole('row').last();
+    this.firstNamaCellLocator = this.lastRowLocator.getByRole('cell').nth(0);
```

Comment:
There is a typo in the variable name `firstNamaCellLocator`. It should be `firstNameCellLocator` to accurately reflect its purpose.

Reply:
corrected